src/special/webpack.js

```diff
@@ -39,14 +44,46 @@ function getLoaders(deps, loaders) {
     .value();
 }
 
+function parseWebpack1(module, deps) {
+  const loaders = getLoaders(deps, module.loaders);
+  const preLoaders = getLoaders(deps, module.preLoaders);
+  const postLoaders = getLoaders(deps, module.postLoaders);
+  return [...loaders, ...preLoaders, ...postLoaders];
+}
+
+function mapRuleLoaders(module) {
+  return module.rules
+    .filter(rule => rule.loaders)
+    .map(rule => rule.loaders.map(loader => loader));
```

Comment:
Sorry, why do we still need `.map(loader => loader)`?

Reply:
Sorry @lijunle, that was dumb of me :(!